ci/jenkins/utils.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function check_and_upgrade_golang() {
+    export GOROOT=/usr/local/go
+    export PATH=${GOROOT}/bin:$PATH
+    echo "====== Checking installed Golang version ======"
+    golang_minor_version=$(go mod edit -json | jq -r .Go)
+    antrea_golang_version="go$(go mod edit -json | jq -r .Go)"
```

Comment:
there seems to be a repetition here. Even if you want to have both variables, you could define `antrea_golang_version` as `"go${golang_minor_version}"`.

Reply:
Fixed.